source/aws-bootstrap-kit/lib/organizational-unit.ts

```diff
@@ -29,50 +30,17 @@ export class OrganizationalUnit extends Construct {
     constructor(scope: Construct, id: string, props: OrganizationalUnitProps) {
         super(scope, id);
 
+        const ouProvider = OrganizationalUnitProvider.getOrCreate(this);
```

Comment:
I assume it's to support import on duplicate?

Reply:
Just like the accounts, using a singleton (and nested stack), help reduce the number of resources in the stack.